dockers/docker-fpm-frr/base_image_files/TSA

```diff
@@ -7,7 +7,18 @@ if [ "$EUID" -ne 0 ] ; then
 fi
 
 if [ -f /etc/sonic/chassisdb.conf ]; then
-  rexec all -c "sudo TSA chassis"
+  CHASSIS_TSA_STATE_UPDATE="CHASSIS_APP_DB HMSET "BGP_DEVICE_GLOBAL\|STATE" tsa_enabled "true""
+  CONFIG_DB_TSA_STATE_UPDATE='{"BGP_DEVICE_GLOBAL":{"STATE":{"tsa_enabled": "true"}}}'
+  current_tsa_state="$(sonic-cfggen -d -v BGP_DEVICE_GLOBAL.STATE.tsa_enabled)"
+  if [[ $current_tsa_state  == true ]]; then
+    echo "Chassis is already in Maintenance"
+    logger -t TSA -p user.info "Chassis is already in Maintenance"
+  else
+    sonic-db-cli $CHASSIS_TSA_STATE_UPDATE
+    sonic-cfggen -a "$CONFIG_DB_TSA_STATE_UPDATE" -w
+    echo "Chassis Mode: Normal -> Maintenance"
+    logger -t TSA -p user.info "Chassis Mode: Normal -> Maintenance"
+  fi
   echo "Please execute \"rexec all -c 'sudo config save -y'\" to preserve System mode in Maintenance after reboot\
```

Comment:
Should this be changed to just 'sudo config save' instead of rexec all now?

Reply:
@fountzou : Please address this comment.

Reply:
Thank you for comment @tjchadaga @abdosi . Fixed -- kindly see https://github.com/sonic-net/sonic-buildimage/pull/19527